### DISCUSSION TITLE
Icon in workflow-templates is not shown

### DISCUSSION THREAD

**Discussion by rymsha** [community/community#25626] [Actions]
<p>I’m following documentation<br>
<a href="https://help.github.com/en/actions/configuring-and-managing-workflows/sharing-workflow-templates-within-your-organization" class="onebox" target="_blank" rel="nofollow noopener">https://help.github.com/en/actions/configuring-and-managing-workflows/sharing-workflow-templates-within-your-organization</a></p>
<p>But can’t make the icon part to work</p>
<blockquote>
<p><code>iconName</code>  -  <strong>Required.</strong>  Defines an icon for the workflow’s entry in the template list. The  <code>iconName</code>  must be an SVG icon of the same name, and must be stored in  <code>.github</code> . For example, a SVG file named  <code>example-icon.svg</code>  is referenced as  <code>example-icon</code> .</p>
</blockquote>
<p>I put the <code>example-icon.svg</code> into root of <code>.github</code> repository, in <code>.github</code> folder of<code>.github</code> repository in <code>icons</code> folder of <code>.github</code> repository. Result is the same - the icon is not shown.</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/rymsha">@rymsha</a>,</p>
<p>Glad to see you in Github Community Forum!</p>
<p>Please put the example-icon.svg in the <code>.github/workflow-templates/</code>  folder.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/81cb9a38d12d6a7f5165a0bdf46bb79a54a163e8.png" data-download-href="/uploads/short-url/iwdOdbniKVfBO6fZViVdJqMpwMg.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181092254-6c8c4d68-7013-4798-95d4-805631c5495c.png" alt="image" data-base62-sha1="iwdOdbniKVfBO6fZViVdJqMpwMg" width="690" height="254" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/81cb9a38d12d6a7f5165a0bdf46bb79a54a163e8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">705×260 12.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>When you create a workflow, it will display the icon:<br>
<img src="https://user-images.githubusercontent.com/52772761/181092255-ab539b73-8c24-4889-80ad-f4fe004410ef.png" alt="image" data-base62-sha1="hDjDfSE5zBzUBsKCmjHgTbDmO7d" width="434" height="250"></p>
<p>Please check my template repository for your reference.<br>
</p><aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/WadeOrg/.github/tree/master/workflow-templates" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/52772761/181092256-cabf12dc-88fd-4ae3-a506-134955074834.png" class="thumbnail onebox-avatar" width="420" height="420">

<h3><a href="https://github.com/WadeOrg/.github/tree/master/workflow-templates" target="_blank" rel="nofollow noopener">WadeOrg/.github</a></h3>

<p>Contribute to WadeOrg/.github development by creating an account on GitHub.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Thanks</p>

**Comment by david-chaves-kiru** [comment]
<p>I’m having the exactly same issue and my repo its’s configured as <a class="mention" href="/weide-zhou">@weide-zhou</a> indicates<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/f/3fb5f27f31a149892f0ca01fabfe4e2a4f616da2.png" data-download-href="/uploads/short-url/95BTQNbcd5vveZmuB8mK6vKltOW.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/85177362/181092257-b445daab-0b88-455f-ae96-4ff685dd73b1.png" alt="image" data-base62-sha1="95BTQNbcd5vveZmuB8mK6vKltOW" width="690" height="295" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/f/3fb5f27f31a149892f0ca01fabfe4e2a4f616da2_2_690x295.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/f/3fb5f27f31a149892f0ca01fabfe4e2a4f616da2_2_1035x442.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/f/3fb5f27f31a149892f0ca01fabfe4e2a4f616da2.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/f/3fb5f27f31a149892f0ca01fabfe4e2a4f616da2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1230×527 36.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
